src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DacFxDeployRequest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
+{
+    /// <summary>
+    /// Parameters for a DacFx deploy request.
+    /// </summary>
+    public class DacFxDeployParams : IScriptableRequestParams
```

Comment:
I don't think DacFx operations are scriptable, so we shouldn't have to implement this interface.

Reply:
This is necessary for the TaskService, since it uses IScriptableRequestParams. Can be fixed in another code review